contrib/DockerFileBuildHelper/Program.cs

```diff
@@ -544,6 +544,12 @@ private DockerInfo GetDockerInfo(Image image)
                     dockerInfo.GitRef = $"v{image.Tag.Split('-')[0]}";
                     dockerInfo.SupportedByUs = true;
                     break;
+                case "thunderhub":
+                    dockerInfo.DockerFilePath = $"Dockerfile";
+                    dockerInfo.GitLink = "https://github.com/apotdevin/thunderhub";
+                    dockerInfo.GitRef = $"{image.Tag.Split('-')[0]}";
```

Comment:
Are you sure about this? The git tags you have start with a `v`

Reply:
The docker hub image also has the `v` so actually leaving it in gives me a double v and the following warning appears: `Broken link detected for image apotdevin/thunderhub:v0.3.6 (https://raw.githubusercontent.com/apotdevin/thunderhub/vv0.3.6/Dockerfile)`

Should work like this. Also don't know if it's good practice to have the docker hub image tag without the `v` but I just left it the same as github repo 😅